website/docs/intro-ee.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Enterprise Edition
+hide_title: true
+---
+import TierLabel from "./_components/TierLabel";
+
+# Weave GitOps Enterprise Edition <TierLabel tiers="Enterprise" />
+
+:::tip Ready for more GitOps?
+To purchase entitlement to Weave GitOps Enterprise Edition please contact [sales@weave.works](mailto:sales@weave.works)
+:::
+
+WGEE provides ops teams with an easy way to assess the
+health of multiple clusters in a single place. It shows cluster information such as
+Kubernetes version and number of nodes and provides details about the GitOps operations
+on those clusters, such as Git repositories and recent commits. Additionally, it
+aggregates Prometheus alerts to assist with troubleshooting.
+
+If you have already purchased your entitlement, head to the [installation page](./installation/weave-gitops-enterprise/index.mdx).
+
+## Feature breakdown
+
+In addition to the features in the OSS edition, Weave GitOps Enterprise Edition
+offers the following capabilitys, taking your delivery from simple CD to IDP:
```

Comment:
```suggestion
offers the following capabilities, taking your delivery from simple CD to IDP:
```

Reply:
Good spot, I took this from the weave.works site, will open an issue there 👍